src/connectionController.ts

```diff
@@ -603,10 +606,12 @@ export default class ConnectionController {
       'mdb.isAtlasStreams',
       false
     );
-    void vscode.window.showInformationMessage('MongoDB disconnected.');
 
     this._disconnecting = false;
-    this._statusView.hideMessage();
+    this._statusView.showMessage('MongoDB disconnected.');
+    setTimeout(() => {
+      this._statusView.hideMessage();
```

Comment:
Should we check that the message that's shown here is still the message we showed initially (if it's overridden we don't want the new message to be hidden instantly because this timer is about to hit).
```suggestion
      const message = 'MongoDB disconnected.';
      this._statusView.showMessage(message);
      if (this._statusView._statusBarItem.text === message) {
        this._statusView.hideMessage();
      }
```